src/workerd/io/compatibility-date.capnp

```diff
@@ -224,7 +224,8 @@ struct CompatibilityFlags @0x8f8c1b68151b6cef {
   # Obsolete flag. Has no effect.
 
   webSocketCompression @20 :Bool
-      $compatEnableFlag("web_socket_compression");
+      $compatEnableFlag("web_socket_compression")
```

Comment:
You also need a `$compatDisableFlag()` so that people can disable this after the default-on date.

Reply:
`no_web_socket_compression` perhaps?